src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/variabledeclarationusagedistance/InputVariableDeclarationUsageDistanceGeneral.java

```diff
@@ -1071,4 +1071,44 @@ private void n() {
         New1.field = (int)a;
     }
 
+    private void o() {
```

Comment:
Better to keep file size in reasonable size  to not scroll to much to top to see config.
Can we create new file for such code ?

Reply:
I recommend we see if we can enforce something through CI. Sounds like a good GSoC simple issue.

Reply:
Done.